net/tcp.lisp

```diff
@@ -760,8 +760,10 @@ to wrap around logic"
                     (challenge-ack connection)))
                ((logtest flags +tcp4-flag-syn+)
                 (challenge-ack connection))
-               ((logtest flags +tcp4-flag-ack+)
-                (detach-tcp-connection connection))))
+               ((not (logtest flags +tcp4-flag-ack+))) ; Ignore packets without ACK set.
```

Comment:
Typo in the commit message for this one.  Not commenting on the logic in the change.